serfclient/client.py

```diff
@@ -74,3 +74,11 @@ def stats(self):
         Obtain operator debugging information about the running Serf agent.
         """
         return self.connection.call('stats')
+
+    def shutdown(self):
```

Comment:
One of the nice things about being consistent and using `close` here is that we can make sure we close any connections in our tests using a `contextmanager`: https://docs.python.org/2/library/contextlib.html#contextlib.closing